lizmap/modules/lizmap/classes/lizmapRepository.class.php

```diff
@@ -147,8 +183,11 @@ public function getProjects()
                     $proj = null;
                     if (in_array($qgsFile.'.cfg', $cfgFiles)) {
                         try {
-                            $proj = lizmap::getProject($this->key.'~'.substr($qgsFile, 0, -4));
+                            // Avoid memory usage by not keeping project instance in projectInstances
+                            $keepReference = false;
+                            $proj = $this->getProject(substr($qgsFile, 0, -4), $keepReference);
```

Comment:
if you modify https://github.com/drillinP/lizmap-web-client/blob/backport-2199-to-release_3_3/lizmap/modules/lizmap/classes/lizmap.class.php#L280
be sure to use 
`lizmap::getProject(substr($qgsFile, 0, -4), $keepReference)` instead

Reply:
@mdouchin the methode `getProject` is added in the commit